exo-create/features/steps/steps-when.ls

```diff
@@ -10,7 +10,7 @@ require! {
 
 module.exports = ->
 
-  @When /^executing "([^"]*)"$/ (command, done) ->
+  @When /^executing "([^"]*)"$/, timeout: 20_000, (command, done) ->
```

Comment:
Turns out I had `node_modules` installed in each of the service templates. I've made a PR to address this: https://github.com/Originate/exosphere/pull/157